src/uct/ib/mlx5/dc/dc_mlx5_ep.h

```diff
@@ -266,6 +267,19 @@ uct_dc_mlx5_iface_is_dci_shared(const uct_dc_mlx5_iface_t *iface)
     return iface->tx.policy >= UCT_DC_TX_POLICY_SHARED_FIRST;
 }
 
+static UCS_F_ALWAYS_INLINE int
+uct_dc_mlx5_iface_is_dcs_quota_or_hybrid(const uct_dc_mlx5_iface_t *iface)
+{
+    return UCS_BIT(iface->tx.policy) & (UCS_BIT(UCT_DC_TX_POLICY_DCS_QUOTA) |
+                                        UCS_BIT(UCT_DC_TX_POLICY_DCS_HYBRID));
+}
+
+static UCS_F_ALWAYS_INLINE int
+uct_dc_mlx5_iface_is_hybrid(const uct_dc_mlx5_iface_t *iface)
+{
+    return iface->tx.policy == UCT_DC_TX_POLICY_DCS_HYBRID;
+}
```

Comment:
I'd do this:
```suggestion
static UCS_F_ALWAYS_INLINE int
uct_dc_mlx5_iface_is_hybrid(const uct_dc_mlx5_iface_t *iface)
{
    return iface->tx.policy == UCT_DC_TX_POLICY_DCS_HYBRID;
}

static UCS_F_ALWAYS_INLINE int
uct_dc_mlx5_iface_is_dcs_quota_or_hybrid(const uct_dc_mlx5_iface_t *iface)
{
    return (iface->tx.policy == UCT_DC_TX_POLICY_DCS_QUOTA) ||;
            uct_dc_mlx5_iface_is_hybrid(iface);
    }

```

Reply:
@yosefe wanted optimize that check as its being called in fast path

Reply:
ok